### PR TITLE
Normalize settings.js naming

### PR DESCRIPTION
**Pull request by jwalgran**
NOTE: This PR implements my preference and does not have to be merged.

The template is named settings.js, and I often look for the wrong name when searching for the view or url that uses it. This commit removes the ambiguity, changing the url to /settings.js and all reference in Python to settings_js.

### CODE REVIEW

File: src/nyc_trees/apps/core/templates/base.html [hunk 1]
```diff
@@ -1,5 +1,5 @@
-{% load utils %}
```
Comment (steventlamb) [inline src/nyc_trees/apps/core/templates/base.html]: why this change?

**Comment by steventlamb** [comment]
+1

**Comment by RickMohr** [comment]
Fine with me to change

**Comment by kdeloach** [comment]
+1

**Comment by maurizi** [comment]
+1

**Comment by jwalgran** [comment]
Closed in favor of #295
